src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from '../TodoItem/TodoItem';
+
+function TodoList({ preparedTodos }) {
+  return (
+    preparedTodos.map(item => (
+      <TodoItem key={item.id} {...item} />
+    ))
+
+  );
+}
+
+TodoList.propTypes = {
+  tasks: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      title: PropTypes.string,
+      completed: PropTypes.bool,
```

Comment:
```suggestion
      name: PropTypes.string.isRequired,
      title: PropTypes.string.isRequired,
      completed: PropTypes.bool.isRequired,
```
You also missed a user field with name in it